src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string | null;
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    key: null,
```

Comment:
Let's use falsy values of same type for initial state values

Reply:
It was already done 😳
![image](https://user-images.githubusercontent.com/121887457/224271245-230f6acc-0a77-468c-802b-929c81a71eeb.png)
![image](https://user-images.githubusercontent.com/121887457/224271285-0ce38490-c84f-4a46-be2c-832cf4c11b03.png)